tests/fixtures.py

```diff
@@ -4110,6 +4110,39 @@
     },
 ]
 
+VOUCHERS = [
+    {
+        "_id": "657e370a4543a555901865c7",
+        "admin_name": "Admin",
+        "code": 7470075124,
+        "create_time": 1702770442,
+        "duration": 480,
+        "for_hotspot": "false",
+        "note": "auto-generated",
+        "qos_overwrite": "false",
+        "quota": 1,
+        "site_id": "5a32aa4ee4b0412345678910",
+        "status": "VALID_ONE",
+        "status_expires": 0,
+        "used": 0,
+    },
+    {
+        "_id": "61facea3873fdb075ce28d71",
+        "admin_name": "Admin",
+        "code": 4470374222,
+        "create_time": 1643826851,
+        "duration": 5184000,
+        "for_hotspot": "false",
+        "note": "auto-generated",
+        "qos_overwrite": "false",
+        "quota": 0,
+        "site_id": "5a32aa4ee4b0412345678910",
+        "status": "USED_MULTIPLE",
+        "status_expires": 244679302,
+        "used": 2,
+    },
+]
```

Comment:
```suggestion
VOUCHERS = [
    {
        "_id": "657e370a4543a555901865c7",
        "admin_name": "Admin",
        "code": 7470075124,
        "create_time": 1702770442,
        "duration": 480,
        "for_hotspot": False,
        "note": "auto-generated",
        "qos_overwrite": False,
        "quota": 1,
        "site_id": "5a32aa4ee4b0412345678910",
        "status": "VALID_ONE",
        "status_expires": 0,
        "used": 0,
    },
    {
        "_id": "61facea3873fdb075ce28d71",
        "admin_name": "Admin",
        "code": 4470374222,
        "create_time": 1643826851,
        "duration": 5184000,
        "for_hotspot": False,
        "note": "auto-generated",
        "qos_overwrite": False,
        "quota": 0,
        "site_id": "5a32aa4ee4b0412345678910",
        "status": "USED_MULTIPLE",
        "status_expires": 244679302,
        "used": 2,
    },
]
```
Please double check with real data, json data when read out typically shows true false without capital letters but in python they should be changed to True/False